pg_hint_plan.c

```diff
@@ -2983,7 +2988,10 @@ get_current_hint_string(ParseState *pstate, Query *query)
 		 * use..
 		 */
 		if (current_hint_str)
-			pfree((void *)current_hint_str);
+		{
```

Comment:
This change does not matter, because we'd just assign back a few lines below and a memory context switch will not fail.